qa/tasks/cephfs/mount.py

```diff
@@ -166,10 +166,10 @@ def _setup_netns(self):
         # Add one new netns and set it id
         args = ["sudo", "bash", "-c",
                 "ip netns add {0}".format(self.netns_name)]
-        self.client_remote.run(args=args, timeout=(5*60))
+        self.client_remote.run(args=args, timeout=(5*60), omit_sudo=False)
```

Comment:
I think since the netns is created with `sudo` and we must set `omit_sudo=False` when mounting. Or the `nsenter` cmd will fail too, with error:

`nsenter: failed to execute XXX: Permission denied`

Reply:
FUSE mounts are created with `sudo` in master too. See - https://github.com/ceph/ceph/blob/master/qa/tasks/vstart_runner.py#L745.

Reply:
Yeah it is. But it will with `omit_sudo=True` as default in `remote.run(..., omit_sudo=True)`, right ?

Reply:
Hmm, right. But I've set `omit_sudo` to `False` already.

Reply:
See - https://github.com/ceph/ceph/pull/35664#discussion_r443556798